src/pages/developing/web-components-tutorial/step-4.mdx

```diff
@@ -31,11 +32,10 @@ A [preview]() of what you'll build:
 <Preview
   height="400"
   title="Carbon for IBM.com Tutorial Step 4"
-  src=""
+  src="https://carbon-design-system.github.io/carbon-for-ibm-dotcom-tutorial/step-4/"
```

Comment:
gotta point to step 5 instead, and add the link to the github page in the `[preview]()` link above!